opentech/apply/views.py

```diff
@@ -1,3 +1,60 @@
-# from django.shortcuts import render
+from django import forms
+from django.template.response import TemplateResponse
 
-# Create your views here.
+from .workflow import SingleStage, DoubleStage
+
+
+workflows = [SingleStage, DoubleStage]
+
+logs = []
+
+submission: dict = {}
+
+
+class BasicSubmissionForm(forms.Form):
+    who_are_you = forms.CharField()
+
+
+def demo_workflow(request, wf_id):
+    wf = int(wf_id)
+    workflow_class = workflows[wf - 1]
+    workflow = workflow_class([BasicSubmissionForm] * wf)
+
+    current_phase = request.POST.get('current')
+    current = workflow.current(current_phase)
+
+    if request.POST:
+        if current.stage.name not in submission:
+            submitted_form = current.stage.form(request.POST)
+            if submitted_form.is_valid():
+                submission[current.stage.name] = submitted_form
```

Comment:
I think ``session``/``logs`` would be better put on ``request.session`` rather than global variables